Resources/Locale/en-US/reagents/meta/consumable/drink/alcohol.ftl

```diff
@@ -157,8 +157,8 @@ reagent-desc-hooch = Either someone's failure at cocktail making or attempt in a
 reagent-name-iced-beer = iced beer
 reagent-desc-iced-beer = A beer which is so cold the air around it freezes.
 
-reagent-name-irish-car-bomb = irish car bomb
-reagent-desc-irish-car-bomb = A troubling mixture of irish cream and ale.
+reagent-name-irish-slammer = irish slammer
+reagent-desc-irish-slammer = A troubling mixture of irish cream and ale.
```

Comment:
this still references the troubles

Reply:
I thought it was probably oblique enough to ignore, as it is a pretty troubline mixture, guiness, whiskey and cream... Any suggestion for a short alternative description?